### PR TITLE
f3d 1.2.1

### PR DESCRIPTION
**Pull request by Meakk**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/f3d.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   desc "Fast and minimalist 3D viewer"
-  homepage "https://kitware.github.io/F3D/"
-  url "https://gitlab.kitware.com/f3d/f3d/-/archive/v1.1.0/f3d-v1.1.0.tar.gz"
-  sha256 "93aa9759efcc4e77beac4568280aaeaca21bfb233d3c9f60262207ca595bde79"
+  homepage "https://f3d-app.github.io/f3d/"
+  url "https://github.com/f3d-app/f3d/archive/refs/tags/v1.2.0.tar.gz"
+  sha256 "a4750217036a47ad26f5a2a77ec40e4b6415290bb55f6f931293ca24d975fa17"
   license "BSD-3-Clause"
-  revision 3
+  revision 1
```
Comment (SMillerDev) [inline Formula/f3d.rb:7]: This should be 0 for new versions, which is the default
```suggestion
```

File: Formula/f3d.rb [hunk 2]
```diff
@@ -23,19 +25,21 @@ class F3d < Formula
 
   fails_with gcc: "5" # vtk is built with GCC
 
-  # Fix build with vtk 9.1.
-  # https://gitlab.kitware.com/f3d/f3d/-/commit/816b09c1e95622d6dc0384cd544572f73deed12c
+  # Fix build
```
[the remark this answers is not in the document]
Comment (Meakk) [reply Formula/f3d.rb:28]: We are gathering issues reported by packagers with 1.2.0 and we plan to release a 1.2.1 later (probably next week)

[the remark this answers is not in the document]
Comment (SMillerDev) [reply Formula/f3d.rb:27]: This should be something like this then.
```suggestion
  # Fixes <specific problem>, can be removed in 1.2.1
```

[the remark this answers is not in the document]
Comment (mwestphal) [reply Formula/f3d.rb:27]: When F3D release a new patch version, this was catched after the 1.2.0 release sadly.

**Comment by chenrui333** [comment]
linux build failure due to missing the bottle for `opencascade`

**Comment by Meakk** [comment]
I'm confused. Why the `opencascade` bottle is missing on Linux? Is it expected?  
Should I disabled the OpenCASCADE module of f3d on Linux? If yes, how can I specify different CMake options depending on the platform?

**Comment by SMillerDev** [comment]
It hasn't been build yet. I'll try and trigger a build for Linux so we can use it here

**Comment by mwestphal** [comment]
@SMillerDev how can we help to move this forward ?

**Comment by Meakk** [comment]
Updated for 1.2.1  
We are still having the issue with Linux.  
@SMillerDev any update?

**Comment by Meakk** [comment]
@SMillerDev 

> /tmp/opencascade-20211214-12902-18biih6/occt-V7_6_0/src/Font/Font_FontMgr.cxx:93:39: fatal error: fontconfig/fontconfig.h: No such file or directory

I guess `fontconfig` dependency is missing from the `opencascade` formula, right?  
I can submit a PR to fix it if you want.

**Comment by SMillerDev** [comment]
That would be great!

**Comment by Meakk** [comment]
`opencascade` has been updated, the bottle should be available for Linux now. How to rerun the CI?

**Comment by SMillerDev** [comment]
Triggered a rebuild, thanks for the effort here @Meakk !

**Comment by Meakk** [comment]
Not sure what is the `tap_syntax` failure. Can someone help?

**Comment by chenrui333** [comment]
the style audit failures have nothing to do with your change, I think a rebase would do the job. Let me do that.

**Comment by chenrui333** [comment]
Please dont use `master` branch for your future PRs, it is pretty hard to rebase and forced update your branch.

**Comment by chenrui333** [comment]
Created a new PR instead, #92288

**Pull request closed without merging**